### DISCUSSION TITLE
Readme profile

### DISCUSSION THREAD

**Discussion by sureshkumar2704** [community/community#133836] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How i can create the readme profile ?

**Comment by kavin-raja-d** [comment]
Make sure to create to readme file with same as your username.

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/133837#discussioncomment-10247376) for reference to why this discussion was closed and locked. Thanks!
